@WillAyd have you seen this failure in the CI on other PRs?

```
=================================== FAILURES ===================================
_______________ TestDatetimeIndex.test_stringified_slice_with_tz _______________
[gw0] darwin -- Python 3.5.6 /Users/vsts/miniconda3/envs/pandas-dev/bin/python

self = <pandas.tests.indexes.datetimes.test_datetime.TestDatetimeIndex object at 0x13accba58>

    def test_stringified_slice_with_tz(self):
        # GH#2658
        import datetime
        start = datetime.datetime.now()
>       idx = date_range(start=start, freq="1d", periods=10, tz='US/Eastern')

pandas/tests/indexes/datetimes/test_datetime.py:105: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
pandas/core/indexes/datetimes.py:1551: in date_range
    closed=closed, **kwargs)
pandas/core/arrays/datetimes.py:454: in _generate_range
    tz, ambiguous=ambiguous, nonexistent=nonexistent)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

>   raise pytz.NonExistentTimeError(stamp)
E   pytz.exceptions.NonExistentTimeError: 2019-03-10 02:01:53.911138

pandas/_libs/tslibs/conversion.pyx:1100: NonExistentTimeError
```

--- a/pandas/core/generic.py
+++ b/pandas/core/generic.py
@@ -6618,10 +6618,10 @@ class NDFrame(PandasObject, SelectionMixin):
             * 'pad': Fill in NaNs using existing values.
             * 'nearest', 'zero', 'slinear', 'quadratic', 'cubic', 'spline',
               'barycentric', 'polynomial': Passed to
-              `scipy.interpolate.interp1d`. Both 'polynomial' and 'spline'
-              require that you also specify an `order` (int),
-              e.g. ``df.interpolate(method='polynomial', order=5)``.
-              These use the numerical values of the index.
+              `scipy.interpolate.interp1d`. These methods use the numerical
+              values of the index.  Both 'polynomial' and 'spline' require that
+              you also specify an `order` (int), e.g.
+              ``df.interpolate(method='polynomial', order=5)``.
             * 'krogh', 'piecewise_polynomial', 'spline', 'pchip', 'akima':
               Wrappers around the SciPy interpolation methods of similar
               names. See `Notes`.

